Tests/Marketplace/marketplace_services.py

```diff
@@ -3929,6 +3934,11 @@ def get_last_commit_from_index(service_account):
     return index_json.get('commit')
 
 
+def get_graph_entity_by_path(entity_path: Path, marketplace):
```

Comment:
Need to make one more method for `get_id_set_entity_by_display_name` for getting the entity by display name.

Reply:
Where do we use it?

Reply:
The name of the method was changed today, it's now `is_content_item_in_id_set` and we use it when filtering by thecontent item display name.